daisy/workflow/workflow.go

```diff
@@ -64,6 +65,28 @@ func (l *gcsLogger) Write(b []byte) (int, error) {
 	return n, err
 }
 
+func daisyBkt(ctx context.Context, client *storage.Client, project string) (string, error) {
+	dBkt := project + "-daisy-bkt"
+	it := client.Buckets(ctx, project)
+	for {
```

Comment:
for bucketAttrs, err := it.Next(); err != iterator.Done; bucketAttrs, err = it.Next() {
}